generators/ReliableStatefulService/templates/gradle/build.gradle

```diff
@@ -4,7 +4,6 @@ apply plugin: 'eclipse'
 sourceSets {
   main {
      java.srcDirs = ['src']
```

Comment:
Let's also update the generator to pull the latest SF java libs

Reply:
done